docs/modules/clients/pages/java.adoc

```diff
@@ -2372,6 +2372,14 @@ increased performance and reduced memory usage.
 to the same member when this property is `true`. When it is set to `false`,
 the client tries to connect to the members in the given order.
 
+|`hazelcast.client.connectivity.logging.delay.seconds`
+|10
+|int
+|Delay in seconds that the client logs connectivity stats on member connection changes.
+The delay functions to reduce noise from frequent connection updates that
+may occur in bursts. Note that the latest connectivity view will be
+logged when the task is run.
```

Comment:
```suggestion
|Delay in seconds before the client logs connectivity statistics after member connection changes.
If a member connection state changes before the delay period has elapsed, the delay will be reset.
The delay exists to reduce noise from frequent connection updates that
may occur in bursts. Note that only the latest connectivity view will be
logged when the task is run after this delay.
```

Reply:
Deleted the comments I was making as James beat me to it! Instead, I'll comment on James' suggestion.....

line 2378: Delay in seconds `after which` the client logs (instead of `before`)?

line 2379: I prefer present tense - `the delay is reset`

line 2380: Can it just be `The delay reduces noise from`...?

line 2381: Again, personal preference - `can `instead of `may `(may implies permission, can just means is able to)

line 2381/2381:: `If the task runs after the configured delay, only the latest connectivity view is logged.`
potentially, `last `instead of `latest`....?